bundlegen/schema/platformSchema.json

```diff
@@ -0,0 +1,310 @@
+{
+    "$schema": "http://json-schema.org/draft/2020-12/schema",
+    "$ref": "#/$defs/Welcome9",
```

Comment:
what's Welcome9 ?

Reply:
This is not seen in the latest schema files.